content/en/monitors/monitor_types/integration.md

```diff
@@ -11,22 +11,92 @@ further_reading:
   text: "Schedule a downtime to mute a monitor"
 - link: "monitors/monitor_status"
   tag: "Documentation"
-  text: "Consult your monitor status"
+  text: "Check your monitor status"
 ---
 
-{{< img src="monitors/monitor_types/integration/es_status.png" alt="es status" responsive="true" >}}
-
 ## Overview
 
-On the integration tab you see a list of your installed integrations. Upon selection, you can choose to monitor either a "Status" or a "Metric".
+Use an integration monitor to check if an installed integration is running. For more detailed monitoring, a metric monitor can be used to gauge specific information about an integration.
+
+## Monitor creation
+
+To create an [integration monitor][1] in Datadog:
+
+1. Use the main navigation: *Monitors --> New Monitor --> Integration*.
+2. Search for an integration or select it from the list or images.
+3. Choose an **Integration Metric** or **Integration Status** monitor:
+    {{< img src="monitors/monitor_types/integration/metric_or_status.png" alt="Metric or Status" responsive="true" style="width:90%;">}}
+
+### Integration Metric
+
+Create an integration metric monitor by following the instructions in the [metric monitor][2] documentation. Using the integration metric monitor type ensures the monitor can be selected by the integration monitor type facet on the [Manage Monitors][3] page.
+
+### Integration Status
+
+If the integration has a service check, the **Integration Status** tab is selectable.
+
+#### Pick a check
```

Comment:
Should we create a numbered list here since it's a set of instructions ? instead of using headers?

Reply:
Nope. I've established a template where all headers in monitor instructions are headers in the docs. This makes it easy for support to link to a particular step.